analysis/webservice/algorithms_spark/DailyDifferenceAverageSpark.py

```diff
@@ -264,7 +267,7 @@ def determine_parllelism(num_tiles):
     return num_partitions
 
 
-def spark_anomolies_driver(tile_ids, bounding_wkt, dataset, climatology, sc=None):
+def spark_anomalies_driver(tile_service_driver, tile_ids, bounding_wkt, dataset, climatology, sc=None):
```

Comment:
Should you name is tile_service_factory as in the other places in the code ?

Reply:
Thank you, I will fix this